features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1772,7 +1772,7 @@
 		    </Claim>
 		</Dialect>
 		<Dialect dialectURI="urn:ietf:params:scim:schemas:core:2.0"
-				 claimURIRegex="urn:ietf:params:scim:schemas:core:2.0:[a-zA-Z0-9$\-_]*$">
+				 claimURIRegex="urn:ietf:params:scim:schemas:core:2.0:[a-zA-Z0-9$\-_.]*$">
```

Comment:
Don't we need `\ ` character for `.` to escape the `.`?

Reply:
resolved by https://github.com/wso2/carbon-identity-framework/pull/3998/commits/71158621d38c5b9d842156c8fa8419a440042c89